External/FEXCore/include/FEXCore/Config/Config.h

```diff
@@ -135,6 +136,29 @@ namespace Type {
       GetListIfExists(Option, &AppendList);
     }
 
+    template <typename TT = T,
+      typename std::enable_if<!std::is_same<TT, std::string>::value, int>::type = 0>
+    Value(FEXCore::Config::ConfigOption _Option)
+      : Option {_Option} {
+      if (!FEXCore::Config::Exists(Option)) {
+        LogMan::Msg::A("FEXCore::Config::Value has no value");
```

Comment:
Should these be `ERROR_AND_DIE` instead of `LogMan::Msg::A`, which gets disabled in release builds?

Reply:
You're right. Let me change over to that